plugins/data-management/src/HierarchicalTrackSelectorWidget/model.ts

```diff
@@ -1,18 +1,32 @@
-import { types, Instance } from 'mobx-state-tree'
+import { types, Instance, addDisposer } from 'mobx-state-tree'
+import { autorun } from 'mobx'
 import {
   getConf,
   readConfObject,
   AnyConfigurationModel,
 } from '@jbrowse/core/configuration'
-import { dedupe, getSession, notEmpty } from '@jbrowse/core/util'
+import {
+  dedupe,
+  getSession,
+  localStorageGetItem,
+  localStorageSetItem,
+  notEmpty,
+} from '@jbrowse/core/util'
 import { ElementId } from '@jbrowse/core/util/types/mst'
 import PluginManager from '@jbrowse/core/PluginManager'
 
 // locals
 import { filterTracks } from './filterTracks'
-import { TreeNode, generateHierarchy } from './generateHierarchy'
+import { generateHierarchy } from './generateHierarchy'
 import { findSubCategories, findTopLevelCategories } from './util'
 
+const localStorageKeyF = () =>
+  typeof window !== undefined
+    ? `distinguished-tracks-${[
+        window.location.host + window.location.pathname,
```

Comment:
I think this should include the config path in it. It may otherwise overwritten by different configs

Reply:
it could be we should create some more generalized way to do this so different use cases are happy with it but I think configPath is something that is done in other cases (example: assemblyselector)